cmd/harmony/main.go

```diff
@@ -645,15 +645,10 @@ func main() {
 		for {
 			select {
 			case sig := <-osSignal:
-				if sig == os.Kill || sig == syscall.SIGTERM {
+				if sig == os.Kill || sig == syscall.SIGTERM || sig == os.Interrupt {
 					fmt.Printf("Got %s signal. Gracefully shutting down...\n", sig)
```

Comment:
Very little time available in interrupt , do the print after

Reply:
Interrupt won't kill the program either way. Better to print the message early to indicate what's happening.

Reply:
the trying to catch on os.Kill is wrong. You can't handle a signal handler on os.kill (signal -9), doesn't work on linux.